app/scripts/controllers/preferences.js

```diff
@@ -55,6 +55,7 @@ export default class PreferencesController {
         autoLockTimeLimit: undefined,
         showFiatInTestnets: false,
         useNativeCurrencyAsPrimaryCurrency: true,
+        defaultHomeActiveTabKey: undefined,
```

Comment:
I'm not sure this is the best place for this state. This isn't really a user preference right now, and if we did want to _make_ it one (e.g. allow the user to configure a default active home tab), we'd still need to store the _last active_ home tab state somewhere for users without that set.

The AppState controller might be a better spot for this.

Reply:
I've updated this to use the `AppStateController`